docs/sources/shared/configuration.md

```diff
@@ -3723,6 +3723,18 @@ shard_streams:
   # CLI flag: -shard-streams.enabled
   [enabled: <boolean> | default = true]
 
+  # Automatically shard streams by adding a __time_shard__ label, with values
+  # calculated from the log timestamps divided by MaxChunkAge/2. This allows the
+  # out-of-order ingestion of very old logs. If both flags are enabled,
```

Comment:
When this says "Both flags" I'm unclear on what the other flag is.  Could you please name it to avoid confusion.

Reply:
The time-sharding feature depends on the stream-sharding to be enabled (directly above in this view). While I agree that this is a bit intransparent in how it is currently worded, I don't think we should change the text in the backport. This change should be introduced into `main` first, to avoid reverting back to the old text in the future.

Reply:
I didn't notice that this was a backport.  I agree, the change should be made in `main`.